pyxem/signals/differential_phase_contrast.py

```diff
@@ -333,24 +333,24 @@ def phase_retrieval(self, method="kottler", mirroring=False, mirror_flip=False):
         References
```

Comment:
Unless the references are really wanted in the source code docstring, I recommend using :cite:`Kottler:08` and so on, and remove the "References" part of the docstring altogether. The docstring in the API reference documentation will then link to the "Bibliography" doc page.